src/components/media-views.js

```diff
@@ -617,6 +622,11 @@ AFRAME.registerComponent("media-video", {
   remove() {
     this.cleanUp();
 
+    const sound = this.el.getObject3D("sound");
```

Comment:
I would do this instead to ensure the THREE.Audio node also gets GCed.

````
if(this.audio) {
  this.el.removeObject3D("sound");
  this.audio.disconnect();
  delete this.audio;
}
```

In practical terms it shouldn't matter either way since if we likely removing this `media-video` component in response to removing the whole entity, but good to always `removeObject3D` for any `setObject3D` you do in a component . (I bet we have other places where we forgot to do this).